src/cms/views/dashboard/redirect_view.py

```diff
@@ -32,9 +32,14 @@ def get(self, request, *args, **kwargs):
         """
 
         user = request.user
+        regions = user.profile.regions
         if user.is_superuser or user.is_staff:
             return redirect("admin_dashboard")
-        regions = user.profile.regions
-        if regions.exists():
+
+        if regions.count == 1:
```

Comment:
```suggestion
        if regions.count() == 1:
```
`count` itself is only the method, and since this method does not equal `1`, this check will always be `False`.

Reply:
Oops!